src/SFA.DAS.LevyTransferMatching.Web/Validators/ValidatorService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using SFA.DAS.LevyTransferMatching.Infrastructure.Services.LocationService;
+using SFA.DAS.LevyTransferMatching.Web.Models.Pledges;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Validators
+{
+    public class ValidatorService : IValidatorService
```

Comment:
I'd add `Location` to the name somewhere, since it's not a generic service but one specific to locations